inspire_json_merger/pre_filters.py

```diff
@@ -62,6 +62,11 @@ def keep_only_update_source_in_field(field, root, head, update):
             ``root`` and ``head``.
     """
     update_sources = {source.lower() for source in get_value(thaw(update), '.'.join([field, 'source']), [])}
```

Comment:
maybe store `thaw(update)` so you don't need to compute it twice?

Reply:
:+1: